tools/install.sh

```diff
@@ -28,11 +28,11 @@ exec 1>$LOGPIPE 2>&1
 CWD=`pwd`
 RootDir=`dirname $CWD`
 
-
 echo "LORIS Installation Script starting at $START"
 
 if [ ! -w $LOGDIR ] ; then
```

Comment:
As I noted above I'm not sure why the `tools/logs/` directory wouldn't be writable since it's created by the install script. However, we can punt this to another issue so that this PR isn't delayed anymore.